### PR TITLE
[experiment] print status-go logs in adb logcat

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -204,3 +206,19 @@ task copyDownloadableDepsToLibs(type: Copy) {
 
 // Must be at bottom to avoid dependency collision
 apply plugin: "com.google.gms.google-services"
+
+task hemroidBuild(type: Exec) {
+    def rootDir = project.rootDir
+    def localProperties = new File(rootDir, "local.properties")
+    def ndkDir = '/Users/romanvolosovskyi/Library/Android/sdk/ndk-bundle'
```
Comment (oskarth) [inline android/app/build.gradle]: Hardcoding this seems bad?

**Comment by oskarth** [comment]
Nice magic, so why does this work? What is this cpp thing?

**Comment by rasom** [comment]
@oskarth that's native code which intercept standard output and redirects it to android's logs.

File: android/app/build.gradle (hunk 1 above)
Comment (rasom) [reply android/app/build.gradle]: fixed

**Review by janherich — APPROVED** [review]

**Comment by oskarth** [comment]
@rasom could add comment to https://codelab.wordpress.com/2014/11/03/how-to-use-standard-output-streams-for-logging-in-android-apps/ just in case someone wants context

**Review by oskarth — APPROVED** [review]

**Pull request merged**